plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/lightgrid/GridColumn.java

```diff
@@ -214,18 +215,32 @@ int computeHeaderWidth() {
         }
         {
             int textWidth;
-            if (Boolean.TRUE.equals(labelProvider.getGridOption(IGridLabelProvider.OPTION_EXCLUDE_COLUMN_NAME_FOR_WIDTH_CALC))) {
-                textWidth = grid.sizingGC.stringExtent("X").x;
-            } else {
-                String text = labelProvider.getText(this);
-                String description = labelProvider.getDescription(this);
-                textWidth = grid.sizingGC.stringExtent(text).x;
-                if (!CommonUtils.isEmpty(description)) {
-                    int descWidth = grid.sizingGC.stringExtent(description).x;
-                    if (descWidth > textWidth) {
-                        textWidth = descWidth;
+            Object calcWidthMethod = labelProvider.getGridOption(IGridLabelProvider.OPTION_CALC_COLUMN_WIDTH_METHOD);
+            String text = "X";
```

Comment:
The `text` variable is not used anywhere by itself. It's only required to calculate the value of the `textWidth` variable. Can you please remove it?